### ISSUE TITLE
Expand the `inspect.py` script

### ISSUE THREAD

**Issue by gbinal** [18F/domain-scan#104]
Expand [that script](https://github.com/18F/domain-scan/blob/make-lambda-optional/scanners/inspect.py), which runs first in the [a11y scan process](https://github.com/18F/domain-scan/wiki/a11y-scanning-process) to have the resulting `inspect.csv` include columns for `Agency` and `Branch`.  

* `Agency` could carry over from the domains.csv file that `inspect.py` is running on.  
* `Branch` - I would think that this could be done using the same method that is applied in this [later script](https://github.com/18F/domain-scan/blob/make-lambda-optional/scripts/a11y/process_a11y.py#L20-L35).  

We would need to ensure that these changes do not adversely impact the workflows for the HTTPS or DAP sections.  cc @konklone 

The benefits to making these changes is that it would make resolving #101 and, to a degree, #102 easier to resolve.

**Comment by konklone** [comment]
This process is best suited for the data loading process in 18f/pulse, not domain scanning. That's already where [branch determination logic](https://github.com/18F/pulse/blob/master/data/processing.py#L789-L813) takes place, and where Agency and Branch are subsequently associated with each domain.

The a11y scanning process should have [`a11y.py`](https://github.com/18F/domain-scan/blob/master/scanners/a11y.py) do all necessary work (calling out to `pa11y` as needed, and synthesizing fields as needed) to produce 1 or more rows' worth of data about a hostname. The Pulse data loading process can then read in that data, associate it with the proper branch and agency, and store the data in Pulse's database alongside HTTPS and DAP data.

The link you put in to inspect.py (https://github.com/18F/domain-scan/blob/make-lambda-optional/scanners/inspect.py) is from a non-`master` branch (the `master` version is identical). But more importantly, inspect.py is no longer used. We now use [`pshtt.py`](https://github.com/18F/domain-scan/blob/master/scanners/pshtt.py), which makes use of [`pshtt`](https://github.com/dhs-ncats/pshtt), an open source collaboration between DHS and GSA. However, as far as I can tell, you shouldn't need to make any changes to `pshtt` to support the `a11y` work.

**Comment by gbinal** [comment]
Thanks - you're right.  Closing out.
